src/EnergyPlus/AirflowNetworkBalanceManager.hh

```diff
@@ -146,12 +146,18 @@ namespace AirflowNetworkBalanceManager {
         Real64 VentilAirTemp;      // Air Temp {C} of ventilation
         Real64 MixVolume;          // Mixing volume of Air {m3}
         Real64 MixMass;            // Mixing mass of air {kg}
+        Real64 ExfilSensiLoss;     // Sensible heat Loss rate {W} due to exfiltration
+        Real64 ExfilLatentLoss;    // Latent heat Loss rate {W} due to exfiltration
+        Real64 ExfilTotalLoss;     // Total heat Loss rate {W} due to exfiltration
+        Real64 ExfilMass;          // Mass of Air {kg} due to exfiltration
+        Real64 InletMass;          // Mass of Air {kg} due to exfiltration
+        Real64 OutletMass;         // Mass of Air {kg} due to exfiltration
```

Comment:
Minor quibble - From the zone's perspective, exfiltration is not really a loss. I would perhaps name these `ExfilSensiHeatEmission` etc. But I see "Loss" has been used for all the new variable names for heat emissions. Then leave it as-is.

The comments on `InletMass` and `OutletMass` are pasteos from `ExfilMass`.